src/main/webapp/app/course/learning-paths/learning-path-management/learning-path-management.component.ts

```diff
@@ -25,12 +26,14 @@ export enum TableColumn {
 @Component({
     selector: 'jhi-learning-path-management',
     templateUrl: './learning-path-management.component.html',
+    styleUrl: 'learning-path-management.component.scss',
 })
 export class LearningPathManagementComponent implements OnInit {
     isLoading = false;
 
     courseId: number;
     health: LearningPathHealthDTO;
+    learningPathsConfiguration: LearningPathsConfiguration;
```

Comment:
I think this could be undefined

```suggestion
    learningPathsConfiguration?: LearningPathsConfiguration;
```